### ISSUE TITLE
Registry-Sweepers Error: contained no hits when hits were expected

### ISSUE THREAD

**Issue by sjoshi-jpl** [NASA-PDS/registry-sweepers#69]
## 💡 Description

The following error has been occurring for ATM and GEO registry-sweeper tasks that has triggered multiple notifications during every Lambda run. Please take a look :

GEO-PROD

```
Error found in log group '/ecs/pds-geo-prod-registry-sweeper-task':

Timestamp (UTC): 2023-09-03 13:46:48.295000
Log Stream: ecs/pds-geo-prod-registry-sweeper-container/8f0a15e6c18d40b59214433c6abbe05f
Error Message: 2023-09-03 13:46:48,295::pds.registrysweepers.utils.db::ERROR::Response for query 346d70 contained no hits when hits were expected.  Returned data is incomplete.  Response was: {'_scroll_id': 'FGluY2x1ZGVfY29udGV4dF91dWlkDnF1ZXJ5VGhlbkZldGNoAxZBSjMxUGlTSFNaS3c1WUd0b0pvZmtBAAAAAAAAPJgWekZYaWNTdFRRX3k3X3NFYWNSRGZlZxZibE9CX0F6SVRyR1R0RHlYSHBNb2F3AAAAAAAAOVEWdW5nTDhrYTJRSmVkdnQzWnAxaDNWZxZBSjMxUGlTSFNaS3c1WUd0b0pvZmtBAAAAAAAAPJcWekZYaWNTdFRRX3k3X3NFYWNSRGZlZw==', 'took': 2, 'timed_out': False, 'terminated_early': False, '_shards': {'total': 3, 'successful': 3, 'skipped': 0, 'failed': 0}, 'hits': {'total': {'value': 3460390, 'relation': 'eq'}, 'max_score': 1.0, 'hits': []}}
```

ATM-PROD

```
Error found in log group '/ecs/pds-atm-prod-registry-sweeper-task':

Timestamp (UTC): 2023-09-03 13:29:26.877000
Log Stream: ecs/pds-atm-prod-registry-sweeper-container/5ab74f8d4b4d45a6b19cda6cfb953956
Error Message: 2023-09-03 13:29:26,877::pds.registrysweepers.utils.db::ERROR::Response for query 945e60 contained no hits when hits were expected.  Returned data is incomplete.  Response was: {'_scroll_id': 'FGluY2x1ZGVfY29udGV4dF91dWlkDnF1ZXJ5VGhlbkZldGNoAxZ3UktlUnNsdVEzU3g2eGx6SVpYaFhRAAAAAAAAACMWdUltaGl1SG9UTU9rSzNWYmo3QTRpdxZ3UktlUnNsdVEzU3g2eGx6SVpYaFhRAAAAAAAAACQWdUltaGl1SG9UTU9rSzNWYmo3QTRpdxZWc180MnowYVFwR2hhb25hd0VncFpRAAAAAAAAAAwWejFZSERmV0xSN2kzclBtNjE1c0c1QQ==', 'took': 1, 'timed_out': False, 'terminated_early': False, '_shards': {'total': 3, 'successful': 2, 'skipped': 0, 'failed': 1, 'failures': [{'shard': -1, 'index': None, 'reason': {'type': 'illegal_state_exception', 'reason': 'node [z1YHDfWLR7i3rPm615sG5A] is not available'}}]}, 'hits': {'total': {'value': 649002, 'relation': 'eq'}, 'max_score': 1.0, 'hits': []}}
```

**Comment by alexdunnjpl** [comment]
Self-note - check whether or not scroll_id is being updated on every request, as it should be

**Comment by alexdunnjpl** [comment]
ATM-PROD appears to be on the OpenSearch side - see failure

```
}
  'shard': -1,
  'index': None,
  'reason': {
    'type': 'illegal_state_exception',
    'reason': 'node [z1YHDfWLR7i3rPm615sG5A] is not available'
  }
}
```

@tloubrieu-jpl @sjoshi-jpl I'm not sure where to take this one, as it appears to be a failure of one of the shards associated with that node.  Any ideas?

GEO-PROD is a bit weirder - it's not showing any shard failures, nor any errors.  I'll introduce some improved logging, hopefully that yields new useful information.

**Comment by jordanpadams** [comment]
@sjoshi-jpl to look at this closer to see if the shard failure is happening regularly.

**Comment by alexdunnjpl** [comment]
Logging improvements introduced in #72 , #73 
https://github.com/NASA-PDS/registry-sweepers/commit/572ac4169bf473ed816d0a5c988009c3516ccc8d
https://github.com/NASA-PDS/registry-sweepers/pull/73/commits/81cb14af7e1882483a980a18086592376a873e9c

**Comment by alexdunnjpl** [comment]
@sjoshi-jpl @jordanpadams I'm fairly certain this is resolved by #77 

Closing on that basis.  @sjoshi-jpl would you please remove any exclusion rules in the error log escalation lambda?  We can re-open if this issue reappears.

**Comment by sjoshi-jpl** [comment]
@alexdunnjpl done. This issue statement has been removed from the sweepers lambda exceptions list and if they do occur again we should know.
